### ISSUE TITLE
Duplicate posts

### ISSUE THREAD

**Issue by albertyw** [albertyw/reaction-pics#81]
https://www.reaction.pics/post/43927587134/junior-hire-working-full-speed
https://www.reaction.pics/post/42091082715/junior-hire-working-full-speed

https://www.reaction.pics/post/52303587985/kindly-assisting-a-new-hire-on
https://www.reaction.pics/post/50978629323/kindly-assisting-a-new-hire-on

**Comment by albertyw** [comment]
Wrote a quick script to dedup everything:

```
import csv

with open('posts.csv', 'r', newline='') as handle:
    reader = csv.reader(handle, dialect=csv.unix_dialect)
    data = list(reader)

print(len(data))
print(data[0])
print(data[0][3])
pics = {}
for row in data:
    if row[3] in pics:
        pics[row[3]].append(row)
    else:
        pics[row[3]] = [row]

for pic, rows in pics.items():
    if len(rows) == 1:
        continue
    titles = [row[1] for row in rows]
    if len(set(titles)) != len(titles):
        row_id = rows[1][0]
        print(row_id)
        data = [r for r in data if r[0] != row_id]

print(len(data))

with open('posts.csv', 'w', newline='') as handle:
    writer = csv.writer(handle, dialect=csv.unix_dialect, quoting=csv.QUOTE_MINIMAL)
    [writer.writerow(row) for row in data]
```

**Comment by albertyw** [comment]
Fixed in b244302
